java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -1083,6 +1084,31 @@ public void lrange_returns_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void lindex_returns_success() {
+        // setup
+        String key = "testKey";
```

Comment:
@acarbonetto 
All keys in the tests must be randomly generated, same as we do in node and python. Sharing the same database between tests can lead to errors if this isn't done.
Please go through all other Java tests and fix that if needed